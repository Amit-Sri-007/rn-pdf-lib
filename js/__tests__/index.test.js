/* @flow */
import * as index from '../../index';

jest.mock('react-native');

describe('index.js', () => {
  it('exports the right stuff', () => {
    const indexKeys = Object.keys(index);
    expect(indexKeys.length).toEqual(3);
    expect(indexKeys).toContain('default');
    expect(indexKeys).toContain('PDFPage');
    expect(indexKeys).toContain('PDFDocument');
    expect(index).toMatchSnapshot();
  });
});
