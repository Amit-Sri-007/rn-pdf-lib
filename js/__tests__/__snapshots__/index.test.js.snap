// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.js exports the right stuff 1`] = `
Object {
  "PDFDocument": [Function],
  "PDFPage": [Function],
  "default": "mock value for PDFLib",
}
`;
